website/docs/components/icon-tile/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,60 @@
+## Usage
+
+### When to use
+
+When displaying an object, or as part of a page title.
+
+### When not to use
+
+Inline in a string of text, such as a paragrph where an object is referred or linked to.
+
```

Comment:
You may want to make the When/When not to use items bullet points for consistency with other docs. (Although since there's just one item each in this case I can understand leaving as is)

Reply:
Just noticed typo: "paragrph" => "paragraph"

Reply:
Fixed the typo, but yes I took the bullets out because they were just single points under each headline.